docs/source/reference/config-proxy.md

```diff
@@ -101,18 +101,18 @@ server {
 If `nginx` is not running on port 443, substitute `$http_host` for `$host` on
 the lines setting the `Host` header.
 
-`nginx` will now be the front facing element of JupyterHub on `443` which means
+`nginx` will now be the front-facing element of JupyterHub on `443` which means
 it is also free to bind other servers, like `NO_HUB.DOMAIN.TLD` to the same port
 on the same machine and network interface. In fact, one can simply use the same
-server blocks as above for `NO_HUB` and simply add line for the root directory
+server blocks as above for `NO_HUB` and simply add a line for the root directory
 of the site as well as the applicable location call:
 
 ```bash
 server {
     listen 80;
     server_name NO_HUB.DOMAIN.TLD;
 
-    # Tell all requests to port 80 to be 302 redirected to HTTPS
+    # Send all requests to port 80 to 302 and redirect to HTTPS
```

Comment:
This configuration line actually returns HTTP code 302 -- it does not redirect traffic from port 80 to port 302.

Reply:
Thank you @ryanlovett, what should be the accurate comment there?

Reply:
There is some information about nginx's `return` directive at https://www.nginx.com/blog/creating-nginx-rewrite-rules/ and about the `302` HTTP code at https://en.wikipedia.org/wiki/HTTP_302. Here, (and above, in the other instance) nginx is redirecting browsers from the current non-HTTPS port 80 to the HTTPS url.

Reply:
@ryanlovett, made changes, kindly review.